landingzones/lz-machinelearning/networking.bicep

```diff
@@ -194,6 +256,14 @@ module nsgAppService '../../azresources/network/nsg/nsg-empty.bicep' = {
 }
 
 // Route Tables
+resource udr 'Microsoft.Network/routeTables@2021-02-01' = {
```

Comment:
Should the udr resource follow the same pattern of nsg with being able to handle an array of udrs?

Reply:
No it shouldn't since the routing will the same for all subnets.